packages/dashboard/src/dashboard.js

```diff
@@ -43,7 +44,12 @@ import { KeyboardOnlyOutline } from './utils';
 import defaultConfig from './defaultConfig';
 
 function Dashboard({ config, children }) {
-  const _config = useMemo(() => deepMerge(defaultConfig, config), [config]);
+  // Need `cloneDeep` so that karma tests are not using a modified
+  // `defaultConfig` between runs in CI
+  const _config = useMemo(
+    () => deepMerge(cloneDeep(defaultConfig), config),
```

Comment:
Does `structuredClone` work? Browser support is really good and there are polyfills

Reply:
TIL about structured clone. very cool reference.

If I'm reading it correctly,  `structuredClone` only works for transferable objects:
https://developer.mozilla.org/en-US/docs/Glossary/Transferable_objects#supported_objects

Also since it moves stuff from one place to another instead of copying it, we'd corrupt `defaultConfig` once the app has mounted once. this would break the `defaultConfig` for future mounts between tests.

Posted this solution above which is a very minor change, doesn't use `deepClone` and eliminates the side effect of mutating `defaultConfig` when we mount the app:
https://github.com/GoogleForCreators/web-stories-wp/pull/10763#discussion_r817930001

Reply:
> The global structuredClone() method creates a [deep clone](https://developer.mozilla.org/en-US/docs/Glossary/Deep_copy) of a given value using the [structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm).

> The method **also** allows [transferable objects](https://developer.mozilla.org/en-US/docs/Glossary/Transferable_objects) in the original value to be transferred rather than cloned to the new object. Transferred objects are detached from the original object and attached to the new object; they are no longer accessible in the original object.

So it works just fine to deep clone a plain old JavaScript objects.

Reply:
Oh so it only moves transferable objects. but functions normally how you'd expect for other objects. thanks for clearing that up!

Reply:
Got rid of `cloneDeep` with Max's suggestion of making `defaultConfig` a function